verilog/tools/ls/BUILD

```diff
@@ -62,11 +62,27 @@ cc_library(
     ],
 )
 
+cc_library(
+    name = "verilog-language-server",
+    srcs = ["verilog-language-server.cc"],
+    hdrs = ["verilog-language-server.h"],
+    visibility = ["//visibility:public"],
+    deps = [
+        ":lsp-parse-buffer",
+        ":verible-lsp-adapter",
+        "//common/lsp:json-rpc-dispatcher",
+        "//common/lsp:lsp-protocol",
+        "//common/lsp:message-stream-splitter",
+        "//common/util:init_command_line",
+    ],
+)
+
 cc_binary(
     name = "verible-verilog-ls",
     srcs = ["verilog_ls.cc"],
     visibility = ["//visibility:public"],
     deps = [
+        ":verilog-language-server",
         ":lsp-parse-buffer",
```

Comment:
a few of these dependencies can probably no go, as they are pulled in from `:verilog-language-server`. So the verible-lsp-adapter, lsp-parse-buffer etc. are not part of the dependencies needed anymore directly in main.

Reply:
I removed excessive `deps` from `verible-verilog-ls` and `verilog-language-server`